config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -42,6 +42,11 @@ data:
       - "Throttling: Rate exceeded"
       installFailingReason: AWSAPIRateLimitExceeded
       installFailingMessage: AWS API rate limit exceeded
+    - name: GeneralQuota
+      searchRegexStrings:
+      - "Quota '[A-Z_]*' exceeded\."
```

Comment:
```suggestion
     - "Quota '[A-Z_]*' exceeded"
```

Reply:
Hmm, doesn't the `-` need to be right under the `s` from the line above?

Also, what's the reason to match on the period?

Reply:
Oh my mistake - I didn't mean to shift the formatting, just wanted to point out we don't need to match the period - fixed the comment